---
layout: post
title: Temporal scaling of aging trajectories and stochastic aging clocks
date: 2025-02-06 12:53:00 -0700
modified: 2025-02-06 12:53:00 -0700
description: Temporal scaling of aging trajectories and stochastic aging clocks
tag:
  - transcriptomic aging clocks
  - temporal scaling
  - long-lived mutants
image: /cara-memperbarui-fork-repository/repo.png
---

Apart from the Gompertz law and the phenomenological observations associated with it [linking the rate of aging and the initial mortality rate](/gompertz_mortality_and_plateau/), there was another interesting observation reported in [(Stroustrup *et al.*, 2016)](https://doi.org/10.1038/nature16550) regarding temporal scaling of aging trajectories in *C.elegans*. Across a lot of environmental stresses and genetic backgrounds, they observed that for most of interventions they used aging curves would simply scale as a function of the rate of aging, thus effectively being determined by a single parameter. In other words, they seem to follow the same aging trajectory with different rates. 

We asked a related question whether those aging trajectories would be accompanied at the molecular level by the same pattern of aging changes, and measured the time series of aging transcriptomes for multiple long-lived strains of C.elegans spanning 10x of wild-type lifespan [(Tarkhov *et al.*, 2019)](https://doi.org/10.1038/s41598-019-43075-z). The term "aging clock" was not widely spread back then, though Horvath's DNA methylation clock had already been published. People say that it was one of the first RNAseq-based aging clocks, though we didn’t use this name back then and referred to it as an "universal transcriptomic biomarker of age". Later, David Meyer used a similar re-branded approach to build a more precise aging clock for *C.elegans* [(Meyer *et al.*, 2021)](https://doi.org/10.1111/acel.13320).

Though, it was kinda cool to see that the aging changes shared the same gene expression signature across 10x variation of lifespan, and the worms simply followed the same path with different aging rates. It indeed hinted at some universality, at least in some sense, of aging across all those strains. Of course, there were some strain-specific changes as well but back then we didn't know what to do with them. We also predicted a few life extending compounds with the help of CMAP — which were supposed to reverse these transcriptomic aging changes. Personally for me, it was one of the first exciting moments in aging research — that we could use computational biology in a predictive way, and that sometimes it would produce some non-trivial predictions, which were extremely hard to come across by "straightforward" high-throughput screens.

In retrospect, now I understand that our first interpretations of those changes as programmatic simply because they were consistent across mutant strains was a bit naive. Phenomenologically, we called it “universal” with some caution not to imply that it was programmatic. Later, David Meyer's simulations [(Meyer *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00619-x) showed, in agreement with our own in the context of epigenetic aging, that purely stochastic changes can lead to the same "universal" aging patterns across multiple conditions. Along the same lines, now I have a better understanding of a weird observation I made back then that, even after correcting for batch effects across different labs, it was extremely hard to pinpoint a single universal aging pattern across different aging studies. It seemed like in every lab the worms would follow a slightly different pattern of aging, which totally makes sense if we assume that those changes are stochastic, and result from a slight drift from a young state — under one set of conditions they would follow one "drift" trajectory, in another they may choose a completely different one. Sometimes, struggling with identifying universal aging genes across multiple experiments may be not a bug but a feature — especially, if aging is indeed predominantly stochastic. If in every new experiment the worms would age slighly differently, it would support the stochastic "fractal hell" scenario for aging.

Interestingly, the DNA methylation changes [(Meyer *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00619-x), [(Tarkhov *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00616-0), [(Tong *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00600-8), [(Bell *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00634-y), the transcriptomic changes [(Tarkhov *et al.*, 2019)](https://doi.org/10.1038/s41598-019-43075-z), [(Meyer *et al.*, 2021)](https://doi.org/10.1111/acel.13320), [(Meyer *et al.*, 2024)](https://doi.org/10.1038/s43587-024-00619-x), and the mortality analyses [(Stroustrup *et al.*, 2016)](https://doi.org/10.1038/nature16550), [(Tarkhov *et al.*, 2017)](https://doi.org/10.1016/j.jtbi.2017.01.017) all together seem to favor the stochastic scenario of aging. Of course, it doesn't mean that there are no programmatic changes at all, neither it means that we cannot reverse those changes with anti-aging interventions, even if the process that led us to the observed aging state was stochastic. Though, it may help improve our conceptual understanding of what aging is and is not.

### References
1. N. Stroustrup, \. \. \., W. Fontana. The temporal scaling of *Caenorhabditis elegans* ageing. *Nature* **530**, 103–107 (2016). [10.1038/nature16550](https://doi.org/10.1038/nature16550)
1. **A.E. Tarkhov**, L.I. Menshikov, P.O. Fedichev. Strehler-Mildvan correlation is a degenerate manifold of Gompertz fit. *J Theor Biol* **416**, 180–189 (2017). [10.1016/j.jtbi.2017.01.017](https://doi.org/10.1016/j.jtbi.2017.01.017)
2. **A.E. Tarkhov**, \. \. \., R.J. Shmookler Reis, P.O. Fedichev. A universal transcriptomic signature of age reveals the temporal scaling of *Caenorhabditis elegans* aging trajectories. *Sci Rep* **9**, 7368 (2019). [10.1038/s41598-019-43075-z](https://doi.org/10.1038/s41598-019-43075-z)
3. D.H. Meyer, B. Schumacher. BiT age: Atranscriptome-based aging clock near the theoretical limit of accuracy. *Aging Cell* **20**, e13320 (2021). [10.1111/acel.13320](https://doi.org/10.1111/acel.13320)
4. **A.E. Tarkhov**, \. \. \., O. Levy, V.N. Gladyshev. Nature of epigenetic aging from a single-cell perspective. *Nat Aging* **4**, 854–870 (2024). [10.1038/s43587-024-00616-0](https://doi.org/10.1038/s43587-024-00616-0)
5. D.H. Meyer, B. Schumacher. Aging clocks based on accumulating stochastic variation. *Nature Aging* **4**, 871–885 (2024). [10.1038/s43587-024-00619-x](https://doi.org/10.1038/s43587-024-00619-x)
6. H. Tong *et al*. Quantifying the stochastic component of epigenetic aging. *Nature Aging* **4**, 886–901 (2024). [10.1038/s43587-024-00600-8](https://doi.org/10.1038/s43587-024-00600-8)
7. C.G. Bell. Quantifying stochasticity in the aging DNA methylome. *Nature Aging* **4**, 755–758 (2024). [10.1038/s43587-024-00634-y](https://doi.org/10.1038/s43587-024-00634-y)
